src/Database/Pivot.php

```diff
@@ -24,58 +25,89 @@ class Pivot extends Model
      *
      * @var string
      */
-    protected $otherKey;
+    protected $relatedKey;
 
     /**
-     * The attributes that aren't mass assignable.
+     * Indicates if the IDs are auto-incrementing.
      *
-     * @var array
+     * @var bool
      */
-    protected $guarded = [];
```

Comment:
This shouldn't have been removed, see https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Eloquent/Relations/Pivot.php#L24